### PR TITLE
Add cox, robustness

### PR DESCRIPTION
**Pull request by oblute**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@rluria14 @ndmaxar agree to maintain?

File: recipes/cox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: baa6efce249e4f696de66df5939066f826cebe4b97379ebab8070cc15c9f812e
+
+build:
+  number: 0
```
Comment (npavlovikj) [inline recipes/cox/meta.yaml:13]: Is there a reason why this recipe is not `noarch: python` ? If some `run` dependencies are missing on some platforms, that is ok - the user will need to manually instal them.

File: recipes/robustness/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82b1cae2a5f16846fd4caadab9cd43fda14f752feec76cc79c38be185255ce99
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```
Comment (npavlovikj) [inline recipes/robustness/meta.yaml:15]: I have the same question about `noarch: python` here is well. I think the recipe should be `noarch`, and the `skip` section can be removed.

File: recipes/robustness/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - cox
+    - dill
+    - gitpython
+    - gputil
+    - grpcio
+    - matplotlib
```
Comment (npavlovikj) [inline recipes/robustness/meta.yaml:28]: Do you need `matplotlib` or `matplotlib-base` ? If the package doesn't depend on interactive images, but only renders images to files, then you can only use `matplotlib-base`. If you need the full Qt backend, then you should use `matplotlib`.

File: recipes/robustness/meta.yaml (hunk 2 above)
Comment (oblute) [reply recipes/robustness/meta.yaml]: @npavlovikj this package doesn't work on osx or win. i've added noarch and removed the skip, but added a comment line to clarify that functionality is not available on those platforms.

Comment (npavlovikj) [reply recipes/robustness/meta.yaml]: That is ok @oblute , thanks.

**Comment by npavlovikj** [comment]
These are noarch recipes, so the Windows failures can be ignored. Thank you for contributing @oblute !

**Pull request merged**